docs/source/notes/broadcasting.rst

```diff
@@ -19,26 +19,26 @@ Two tensors are "broadcastable" if the following rules hold:
 
 For Example::
 
-    >>> x=torch.FloatTensor(5,7,3)
-    >>> y=torch.FloatTensor(5,7,3)
+    >>> x=torch.empty(5,7,3)
+    >>> y=torch.empty(5,7,3)
     # same shapes are always broadcastable (i.e. the above rules always hold)
 
-    >>> x=torch.FloatTensor()
-    >>> y=torch.FloatTensor(2,2)
+    >>> x=torch.empty(( ,0))
+    >>> y=torch.empty((2,2))
```

Comment:
why is this a tuple when all the other ones aren't?

Reply:
I'd maybe encourage writing them all as a tuple, because this is what `numpy.empty` expects?